roles/convert2rhel/README.md

```diff
@@ -0,0 +1,42 @@
+theforeman.foreman.convert2rhel
+===========================
+
+This role creates a basic configuration for everything needed to register and convert `CentOS` clients to `Red hat Enterprise Linux`.
+
+First is upload of manifest & sync of `RHEL 7` & `RHEL 8` repositories, for more detail see [content_rhel Role](https://github.com/theforeman/foreman-ansible-modules/blob/develop/roles/content_rhel/README.md).
+
+The role creates then Convert2RHEL products & repositories (with synchronization), activation keys and host groups for each OS.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `foreman_c2r_simple_content_access`: TODO: Simple content access enabled.
+- `foreman_c2r_manifest_path`: Path to manifest, default: `~/manifest.zip`.
+- `foreman_c2r_enable_rhel7`: Enable & sync RHEL 7 repositories, default: `true`.
+- `foreman_c2r_enable_rhel8`: Enable & sync RHEL 8 repositories, default: `true`.
+- `foreman_c2r_rhel8_releasever`: RHEL 8 repository version, required if `foreman_c2r_enable_rhel8` is `true`.
```

Comment:
is there a reason to have these "redefined" instead of using the defaults of `content_rhel` and let the user override the `conent_rhel` values directly?